utils/build_swift/driver_arguments.py

```diff
@@ -564,6 +564,11 @@ def create_argument_parser():
         '--skip-test-optimized',
         action=arguments.action.unavailable)
 
+    parser.add_argument(
```

Comment:
Would you mind updating this option to use the new style?

```
option('--use-lld-linker', toggle_true,
       help='use lld linker instead of gold for Linux toolchains')
```